src/utils.js

```diff
@@ -15,9 +15,12 @@ export function compose(...funcs) {
 
 // Get the icon URL of an app
 export function appIconUrl(app) {
-  return app && app.baseUrl
-    ? resolvePathname('images/icon.svg', app.baseUrl)
```

Comment:
Ahh shoot, we should also provide a fallback to fetch the old icon so they don't get broken by this change.

Reply:
Ah yes, we could add a `legacyAppIconUrl()` function to do that, and add another `RemoteImage` here, in case the first one is not found: https://github.com/aragon/aragon/blob/master/src/components/AppIcon/AppIcon.js#L59